openai_vector/challenges/run-benchmark.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "run-benchmark",
+    "default": true,
+    "description": "Challenge to close, restore, and benchmark index with specified operations.",
+    "schedule": [
+      {
+        "name": "close-benchmark-index",
```

Comment:
Once the index is closed, the wait-until merge finishes doesn't complete, since there are no indicies being returned.